multiversx_sdk_cli/tests/test_accounts.py

```diff
@@ -1,44 +1,45 @@
 from pathlib import Path
 
 import pytest
+from multiversx_sdk_core import Address, Transaction, TransactionPayload
 
 from multiversx_sdk_cli.accounts import Account
-from multiversx_sdk_cli.transactions import Transaction
 
 
 def test_sign_transaction():
     alice_pem = Path(__file__).parent / "testdata" / "alice.pem"
     alice = Account(pem_file=str(alice_pem))
 
     # With data
-    transaction = Transaction()
-    transaction.nonce = 0
-    transaction.value = "0"
-    transaction.sender = "erd1l453hd0gt5gzdp7czpuall8ggt2dcv5zwmfdf3sd3lguxseux2fsmsgldz"
-    transaction.receiver = "erd188nydpkagtpwvfklkl2tn0w6g40zdxkwfgwpjqc2a2m2n7ne9g8q2t22sr"
-    transaction.gasPrice = 200000000000000
-    transaction.gasLimit = 500000000
-    transaction.data = "foo"
-    transaction.chainID = "chainID"
-    transaction.version = 1
-    transaction.signature = alice.sign_transaction(transaction)
-
-    assert "0e69f27e24aba2f3b7a8842dc7e7c085a0bfb5b29112b258318eed73de9c8809889756f8afaa74c7b3c7ce20a028b68ba90466a249aaf999a1a78dcf7f4eb40c" == transaction.signature
+    transaction = Transaction(
```

Comment:
I think these tests can be removed (the one with signing a transaction), since we have them (most of the flow, at least) in sdk-core, as well. Can stay for now, but we can remove them in the future.

Reply:
Yes, indeed. Will be removed in the future.